README.md

```diff
@@ -287,6 +292,10 @@ options:
   accepts fractional number parts, like `0.3`. A `neg_ttl` of `0` means the
   cached misses will never expire.
   **Default:** inherited from the instance.
+- `l1_serializer`: function used to transform the values before setting them
+  in the LRU cache (L1). It might be useful to store arbitrary Lua objects
+  such as cdata objects, functions, ... This function is called on L1 miss.
+  **Default:** inherited from the instance.
```

Comment:
I was thinking about a more real-life example: compiling lua source code into a callable function.

I think it would be a better example as it is not something that you could do at any other place, also it exhibits better the error management choices that have to be made (as blindly propagating errors could easily result into error storms).

Reply:
Agreed, your example is more appropriate :)